electrode-ota-server-routes-auth/src/index.js

```diff
@@ -145,10 +147,16 @@ export const register = diregister({
                             //(email, createdBy, friendlyName, ttl
                             addAccessKey(email, hostname, displayName || username, void(0), (e, token) => {
                                 if (e) {
-                                    console.error('Error adding key', e);
+                                    let message = 'Error adding access key';
+                                    let href = '';
+                                    if (e.message.indexOf('No user registered') >= 0) {
+                                        message = e.message + " Try using 'code-push register' instead of 'code-push login', or click here to register.";
+                                        href = '/auth/register/' + name;
```

Comment:
this assumes we have a registerPath, which may not be true for all auth methods.